frontend/src/pages/spotnet/dashboard/add-deposit/AddDeposit.jsx

```diff
@@ -0,0 +1,58 @@
+import React, { useState } from 'react';
+import Card from 'components/Card/Card';
+import TokenSelectorDeposit from 'components/TokenSelector/TokenSelector';
+import { ReactComponent as HealthIcon } from 'assets/icons/health.svg';
+import { ReactComponent as EthIcon } from 'assets/icons/ethereum.svg';
+import { validateNumberInput } from '../../../../utils/regex';
+import './AddDeposit.css';
+
+export default function AddDeposit() {
+  const [amount, setAmount] = useState('0');
+  
+  const handleAmountChange = (e) => {
+    const value = e.target.value;
+    if (validateNumberInput(value)) {
+      setAmount(value);
+    }
+  };
+
+  return (
+    <div className="deposit-wrapper">
+      <div className="deposit-container">
+        <h1 className="deposit-title1">zkLend Deposit</h1>
+        <div className="main-container-deposit">
+          <div className="top-cards-deposit">
+            <Card label="Health Factor" value="1.47570678" icon={<HealthIcon className="icon" />} />
+            <Card label="Borrow Balance" value="$-55.832665" icon={<EthIcon className="icon" />} />
+          </div>
+        </div>
+        <h1 className="deposit-title2">Pls make a deposit</h1>
+        <div className="">
+          <label className="token-select-label">Select token</label>
+          <TokenSelectorDeposit />
+        </div>
+
+        <div className="amount-input-deposit" aria-labelledby="amount-input-label">
+          <input
+            type="text"
+            id="amount-field"
+            value={amount}
+            onChange={handleAmountChange}
+            pattern="^\d*\.?\d*$"
+            className="amount-field-deposit"
+            aria-describedby="currency-symbol"
+            placeholder="0.00"
+          />
+          <span id="currency-symbol" className="currency-deposit">
+            STRK
+          </span>
+        </div>
+
+        <div className='dep-button'>
```

Comment:
I don't see any integration with a backend, please refer to the issue.

Reply:
@whateverfw   Pls kindly point to the APIs for the integration

Reply:
https://github.com/djeck1432/spotnet/issues/404
```
6. Send to backend post request /api/add-extra-deposit/{position_id} with position_id and amount, token_symbol
7. Position_id you can fetch in dashboard page hook
```

Reply:
Integration treated in the latest push. Pls checked